src/actors/objects/mod.rs

```diff
@@ -663,16 +586,17 @@ impl ObjectsActor {
     }
 }
 
-enum DownloadStream {
-    FutureStream(Box<dyn Stream<Item = Bytes, Error = ObjectError>>),
-    File(PathBuf),
-}
-
+/// Create a list of all DIF downloads to be attempted.
+///
+/// Builds the list of download locations where the relevant Debug Information
+/// Files for a given [ObjectId] might be relative to the [SourceConfig].  This
+/// does not mean that each file exists on the source.  For some sources,
+/// e.g. [SourceConfig::Sentry], this might involve querying the source.
```

Comment:
as we found out, stable rustdoc does not auto-resolve these links :-(

Reply:
good catch, i tried to update this all but stable rustdoc doesn't even seem to warn when links are broken which makes this really hard to spot :(